internal/service/project/model_project.go

```diff
@@ -16,12 +18,13 @@ func NewTFProjectDataSourceModel(ctx context.Context, project *admin.Group, proj
 	}
 	projectSettings := projectProps.Settings
 	return &TFProjectDSModel{
-		ID:           types.StringValue(project.GetId()),
-		ProjectID:    types.StringValue(project.GetId()),
-		Name:         types.StringValue(project.Name),
-		OrgID:        types.StringValue(project.OrgId),
-		ClusterCount: types.Int64Value(project.ClusterCount),
-		Created:      types.StringValue(conversion.TimeToString(project.Created)),
+		ID:                      types.StringValue(project.GetId()),
+		ProjectID:               types.StringValue(project.GetId()),
+		Name:                    types.StringValue(project.Name),
+		OrgID:                   types.StringValue(project.OrgId),
+		ClusterCount:            types.Int64Value(project.ClusterCount),
+		Created:                 types.StringValue(conversion.TimeToString(project.Created)),
+		RegionUsageRestrictions: types.StringPointerValue(project.RegionUsageRestrictions),
```

Comment:
Can you include the related jira ticket and a brief summary of the change? Is this an attribute defined in the project resource which was not being used?

Reply:
added ticket and updated description.

Reply:
does RegionUsageRestrictions need to be exported in the resource and datasources schema?

Reply:
yes, based on this attribute value it is configured which regions can be configured for this project's clusters, etc, so users should be able to access it. The attribute is already a part of resource and data source schemas, but it's value was never actually being set